tests/System.Text.Primitives.Tests/Formatting/TimeTests.cs

```diff
@@ -51,6 +53,7 @@ public void RandomDateTimeTests(char format)
         static void TestDateTimeFormat(DateTime dt, char format, SymbolTable symbolTable)
         {
             var expected = dt.ToString(format.ToString(), CultureInfo.InvariantCulture);
+            if (format == 'r') expected = expected.ToLowerInvariant();
```

Comment:
'x' and 'X' both produce hex with lower and upper case. I am not saying that ToString should change what it does with 'r'. What I am saying is that in System.Text.Primitives we need the ability to produce lowercased R format, and I thought 'r' would be best for it.